source/api_cc/include/common.h

```diff
@@ -19,6 +19,8 @@
 namespace deepmd {
 
 typedef double ENERGYTYPE;
+// TODO: currently we only implement TF; reserve for future use
+enum Backend { TENSORFLOW, PYTORCH, PADDLE, UNKNOWN };
```

Comment:
It would be better to avoid using magic strings for the backend names. Consider using a map or a set of constants instead. This would make the code more maintainable and less error-prone. [important]

Reply:
I've changed to the lowercase.